decidim-proposals/app/queries/decidim/proposals/similar_proposals.rb

```diff
@@ -30,8 +30,8 @@ def query
           .published
           .where(
             "GREATEST(#{title_similarity}, #{body_similarity}) >= ?",
-            proposal.title,
-            proposal.body,
+            @proposal[:title],
+            @proposal[:body],
```

Comment:
What's the motivation for this change?

Reply:
Previously the `proposal` was created as a `draft` before the `Compare` step (or similar proposal search), with this PR the `proposal` is created after. 
So `:title` and `:body` are form `params`, not instance attributes.